README.md

```diff
@@ -31,41 +32,41 @@ if it's a library, framework or CLI tool.
 
 ### Common Classes
 
-* [pyTooling.CallByRef.*](https://pyTooling.GitHub.io/pyTooling/CallByRef/)  
+* [pyTooling.CallByRef.*](https://pyTooling.GitHub.io/pyTooling/CallByRef/)
   Emulation of *call-by-reference* parameters.
-* [pyTooling.Versioning.*](https://pyTooling.GitHub.io/pyTooling/Versioning/)  
+* [pyTooling.Versioning.*](https://pyTooling.GitHub.io/pyTooling/Versioning/)
   Class representations of semantic version (SemVer) and calendar version (CalVer) numbers.
 
 
 ### Decorators
 
-* [export](https://pyTooling.GitHub.io/pyTooling/Decorators/Visibility.html#export)    
+* [export](https://pyTooling.GitHub.io/pyTooling/Decorators/Visibility.html#export)
   Register the given function or class as publicly accessible in a module.
 
 
 ### Exceptions
 
-* [EnvironmentException](https://pyTooling.GitHub.io/pyTooling/Exceptions/PredefinedExceptions.html#environmentexception)  
```

Comment:
The vscode settings are missing to preserve enforced line-breaks.

Reply:
Fixed in https://github.com/pyTooling/pyTooling/pull/38/commits/3e557eb13da0a4f080d61b43d5b312c86ad4d666.